controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -658,3 +683,26 @@ func (r *StorageConsumerReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Owns(&rookCephv1.CephClient{}).
 		Complete(r)
 }
+
+func (c *cephClientFormat) generateHashForCephClient(storageConsumerName, claimID, cephUserType string) string {
```

Comment:
Why do we need this as a function on cephClientFormat? I think we should hide this structure as much as possible, even just make it an internal structure inside the function itself.

Reply:
done